config/default.yml

```diff
@@ -3020,6 +3020,11 @@ Style/MultilineTernaryOperator:
   Enabled: true
   VersionAdded: '0.9'
 
+Style/MultilineWhenThen:
+  Description: 'Do not use then for multi-line when statement'
```

Comment:
Add `StyleGuide: '#no-then'` if The Ruby Style Guide accepts it.
https://github.com/rubocop-hq/rubocop/blob/v0.71.0/config/default.yml#L3458